### PR TITLE
Improvements(UI): Made table styling consistent across the project

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on making the table styling consistent across the project

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2022-09-02 at 12 22 35 PM
Screenshot 2022-09-02 at 12 22 45 PM
Screenshot 2022-09-02 at 12 22 53 PM
Screenshot 2022-09-02 at 12 23 07 PM
Screenshot 2022-09-02 at 12 23 22 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx [hunk 1]
```diff
@@ -567,10 +567,10 @@ const DashboardDetails = ({
                       />
                     </div>
                   </div>
-                  <div className="tw-table-responsive tw-my-6">
+                  <div className="tw-table-responsive tw-my-6 tw-bg-white tw-border tw-border-main tw-rounded tw-shadow">
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx:570]:
```css
tw-table-responsive tw-my-6 tw-bg-white tw-border tw-border-main tw-rounded tw-shadow
```
As we are using these classes in all places so can we create one common class and use it everywhere?

File: openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx [hunk 2]
```diff
@@ -581,7 +581,7 @@ const DashboardDetails = ({
                         {charts.map((chart, index) => (
                           <tr
                             className={classNames(
-                              'tableBody-row',
+                              'tableBody-row tw-border-l-0 tw-border-r-0 tw-border-b-0',
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx:584]: Can't we directly modify the `tableBody-row` class?

File: openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx]: Made requested changes

File: openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx]: Made requested changes

File: openmetadata-ui/src/main/resources/ui/src/components/DashboardVersion/DashboardVersion.component.tsx [hunk 3]
```diff
@@ -249,10 +249,10 @@ const DashboardVersion: FC<DashboardVersionProp> = ({
                       description={getDashboardDescription()}
                     />
                   </div>
-                  <div className="tw-table-responsive tw-my-6 tw-col-span-full tw-bg-white tw-border tw-border-main tw-rounded tw-shadow">
+                  <div className="tw-table-responsive tw-my-6 tw-col-span-full tw-table-container">
                     <table className="tw-w-full" data-testid="schema-table">
                       <thead>
-                        <tr className="tableHead-row tw-border-0">
+                        <tr className="tableHead-row0">
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/DashboardVersion/DashboardVersion.component.tsx:255]:
```suggestion
                        <tr className="tableHead-row">
```

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**